modules/openapi-generator/src/main/resources/python/api_test.mustache

```diff
@@ -27,8 +39,14 @@ class {{#operations}}Test{{classname}}(unittest.TestCase):
 {{#summary}}
         {{{summary}}}  # noqa: E501
 {{/summary}}
+
```

Comment:
Why not use code in here?
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/api_doc_example.mustache
If you separated api_doc_example.mustache into:
- api_doc_example_imports.mustache
- api_doc_example_code.mustache
Then you could use api_doc_example_code.mustache here
and use the same mustache file in two places

Reply:
Nice idea. Can you provide an example? I did not work on the documentation up to now.

Reply:
Here is an example where a mustache file imports other mustache files into itself
https://github.com/spacether/openapi-generator/blob/allof_anyof_oneof_addition/modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache#L59

Reply:
As you suggested in the comment above, I will focus on the models, and I will correct this in a future PR.